editor/debugger/script_editor_debugger.cpp

```diff
@@ -1683,7 +1683,7 @@ void ScriptEditorDebugger::_item_menu_id_pressed(int p_option) {
 			// Parse back the `file:line @ method()` string.
 			const Vector<String> file_line_number = ci->get_text(1).split("@")[0].strip_edges().split(":");
 			ERR_FAIL_COND_MSG(file_line_number.size() < 2, "Incorrect C++ source stack trace file:line format (please report).");
-			const String file = file_line_number[0];
+			const String &file = file_line_number[0];
```

Comment:
Replace this in the use below, it's used only once

Reply:
I'm biased against this change since I tend to use the same assignment pattern after a `String.split`. I personally think it adds readability.